browser/src/components/FoiaList.js

```diff
@@ -47,13 +72,15 @@ const FoiaList = (props) => { const [status, setStatus] = useState();
           <select id="foia-list-prices" onChange={event => setFilters({...filters, price: event.target.value})}>
             <option value="" key="no-price">Select a price</option>
             {filterData.prices.map(price => (
-              <option value={price} key={price}>{new Intl.NumberFormat(navigator.language, {style: 'currency', currency: 'USD'}).format(price)}</option>
+              <option value={price} key={price}>{new Intl.NumberFormat(navigator.language, {style: "currency", currency: "USD"}).format(price)}</option>
             ))}
           </select>
           <label htmlFor="foia-list-turnaround">Turnaround time:</label>
-          <select id="foia-list-turnaround" onChange={event => setFilters({...filters, datetime_done: event.target.value})}>
-            <option value="">Select a date…</option>
-            <option value="completed">Completed</option>
+          <select id="foia-list-turnaround" onChange={event => setFilters({...filters, turnaroundTime: event.target.value})}>
+            <option value="" key="no-date">Select a date…</option>
+            {filterData.turnaroundTimes.map(time => (
+              <option value={time.value} key={time.value}>{time.label}</option>
```

Comment:
If I change this to: `<option value={time.value} key={time.label}>{time.label}</option>`, it seems to work.
- correctly changes the state to something like `2020-07-19T10:40:02.641-04:00`
- correctly shows something like “Over five months” in the select

I am not confident of my knowledge of `key` w/in React. Is there a bad side effect of this?

Reply:
I don't know why, but this works! Thank you so much.

My knowledge of `key` in React is also limited, so it still doesn't make sense to me why changing `value` to `label` would make a difference, other than it might check the content of the `option` tag somehow?